crates/pop-cli/src/commands/call/chain.rs

```diff
@@ -90,7 +95,7 @@ impl CallChainCommand {
 			};
 
 			// Sign and submit the extrinsic.
-			if let Err(e) = call.submit_extrinsic(&chain.client, xt, &mut cli).await {
+			if let Err(e) = call.submit_extrinsic(&chain.client, &chain.url, xt, &mut cli).await {
```

Comment:
`Client` and `Url` are passed to the function in stead of `Chain` because the latter is a type used in the cli crate only.